cfgen/cfgen

```diff
@@ -783,6 +783,17 @@ Oid.__doc__ = 'Motr conf object identifier'
 Oid.fidk.__doc__ = '.f_key part of the corresponding m0_fid'
 
 
+ObjTMask = NamedTuple('ObjTMask', [('cont', int), ('key', int)])
+ObjTMask.__doc__ = 'Motr conf object identifier mask'
+ObjTMask.key.__doc__ = '.f_key part mask of the corresponding m0_fid'
+
+
+ObjT_Mask_Map = {
+    # here for object key - lower 32 bits is masked for dynamic part. 
+    ObjT.process: ObjTMask(0xffffffffffffffff, 0xffffffff00000000)
```

Comment:
@Shreya-18, let's use low 32 bits for base fid key and high 32 bits for dynamic. This will make all the base fids look the same.

Reply:
Sure @mssawant , Need to notify S3 team with the same. They are considering lower 32bits as dynamic.

Reply:
done